.github/workflows/release.yml

```diff
@@ -165,8 +163,42 @@ jobs:
           name: builds
           path: builds
 
+  build_aarch64_linux:
```

Comment:
This new job is an amalgamation of copy-pasted pieces of `build_linux` and pieces of the micro-RDK's `build-installer-arm64-linux` [job](https://github.com/viamrobotics/micro-rdk/blob/main/.github/workflows/publish.yml#L72-L103). Looks like they ran into a similar issue with `aarch64` and the `ring` crate, and had to push the building out into a separate job; made sense to me to just copy pieces of what they had done. I'll be honest that I don't know _exactly_ what was fixed with these changes, but `aarch64` seems to build fine.